src/content.js

```diff
@@ -148,6 +148,19 @@ async function addTrendingMenuItem() {
 	}
 }
 
+async function addProfileHotkey() {
+	const userMenuExists = select.exists('#user-links');
+	if (!userMenuExists) {
+		return;
+	}
+
+	const menuItem = select.all('#user-links a.dropdown-item').filter(item => {
+		return item.text.trim().toLowerCase() === 'your profile';
+	});
```

Comment:
You can select it directly with `'[href="/${getUsername()}"]'` instead of the previous 7 lines.